### PR TITLE
Feature/zk UI 141 init roles support in the UI

### PR DESCRIPTION
**Pull request by JBWatenbergScality**
- [x] Rename Account userName property to Name for more clarity and avoid confusion with IAM users
 - [x] Fetch rolesForWebIdentity
 - [ ] Refactor account listing and breadcrumb to display loading/errors
 - [ ] Test accounts retrieval --> useAccounts() hook tests

Coming in an upcoming PR
 - [ ] Assume the right role when selecting an account 
 - [ ] Remove accounts fetching and mapping by id (assumeRole just need the role ARN)

### CODE REVIEW

File: src/react/utils/hooks.ts [hunk 1]
```diff
@@ -1,8 +1,14 @@
 import { useEffect, useState } from 'react';
+import { useQuery } from 'react-query';
 import { useSelector } from 'react-redux';
 import { useLocation } from 'react-router-dom';
 import { addTrailingSlash } from '.';
+import { getRolesForWebIdentity } from '../../js/IAMClient';
+import { Account } from '../../types/iam';
 import { AppState } from '../../types/state';
+import { notFalsyTypeGuard } from '../../types/typeGuards';
+import { useIAMClient } from '../IAMProvider';
```
Comment (ChengYanJin) [inline src/react/utils/hooks.ts:10]: could be removed since it's not used

File: src/react/utils/hooks.ts [hunk 2]
```diff
@@ -1,8 +1,14 @@
 import { useEffect, useState } from 'react';
+import { useQuery } from 'react-query';
```
Comment (ChengYanJin) [inline src/react/utils/hooks.ts:2]: could be removed

**Review by ChengYanJin — APPROVED** [review]
LGTM! Just some not used import to remove.

**Comment by JBWatenbergScality** [comment]
Will clean these unused import in next pr, let's close this one !

**Pull request merged**